metaci/build/tasks.py

```diff
@@ -93,14 +94,23 @@ def run_build(build_id, lock_id=None):
 
 
 def start_build(build, lock_id=None):
-    queue = django_rq.get_queue(build.plan.queue)
-    result = queue.enqueue(
-        run_build, build.id, lock_id, job_timeout=build.plan.build_timeout
-    )
-    build.task_id_check = None
-    build.task_id_run = result.id
-    build.save()
-    autoscale()
+    queue_name = build.plan.queue
+    if queue_name == "perftest":
+
+        class Result(T.NamedTuple):
+            id: T.Any
+
+        job_id = run_one_off_build(build)
+        return Result(job_id)
```

Comment:
If there's an error here we should make sure that gets recorded on the build model and its status updated to error

Reply:
Would probably also be good to log what dyno it's running in when it's successful, so that we can find it if we need to kill it.